packages/time-series-chart/src/Data.tsx

```diff
@@ -0,0 +1,13 @@
+export interface typeoptions {
+  chart: {
+    id: string;
+  };
+  xaxis: {
+    categories: Array<string | number>;
+  };
+}
+
+export interface typeseries {
+  name: string;
+  data: Array<number>;
+}
```

Comment:
I would use same type name standard used with Java type and remove the "type" prefix. It could be: Options and Series or ApexOptions or ApexSeries.